apps/ledger-live-desktop/src/renderer/hooks/usePortfolioCards.tsx

```diff
@@ -0,0 +1,69 @@
+import React, { useEffect, useState } from "react";
+import { useDispatch, useSelector } from "react-redux";
+import ActionCard from "~/renderer/components/ContentCards/ActionCard";
+import { portfolioContentCardSelector } from "~/renderer/reducers/dynamicContent";
+import * as braze from "@braze/web-sdk";
+import { setPortfolioCards } from "~/renderer/actions/dynamicContent";
+
+const usePortfolioCards = () => {
+  const dispatch = useDispatch();
+  const [cachedContentCards, setCachedContentCards] = useState<braze.Card[]>([]);
+  const portfolioCards = useSelector(portfolioContentCardSelector);
+
+  useEffect(() => {
+    const cards = braze.getCachedContentCards().cards;
+    setCachedContentCards(cards);
+  }, []);
```

Comment:
Is there a reason why we don't initiate cachedContentCards with `braze.getCachedContentCards().cards`?
This would allow us to remove a `useEffect`

Reply:
I re-used part of the old code and that was how it was done and I didn't give it a second thought. Agreed, nice catch, updating it right now.